README.md

```diff
@@ -1,12 +1,8 @@
 # TapResearch-iOS-SDK
-TapResearch iOS SDK v3.1.0-beta4-14
-
+TapResearch iOS SDK v3.2.0-beta

```

Comment:
does this need a number at the end? v3.2.0-beta4?

Reply:
ah nice - yeah it needs a '2' - i'll add